packages/flutter_markdown/lib/src/markdown_raw.dart

```diff
@@ -129,7 +129,25 @@ class _MarkdownBodyRawState extends State<MarkdownBodyRaw> {
 
   void initState() {
     super.initState();
+    _buildMarkdownCache();
+  }
+
+  void dispose() {
+    _linkHandler.dispose();
+    super.dispose();
+  }
+
+  void didUpdateConfig(MarkdownBodyRaw oldConfig) {
+    if (oldConfig.data != config.data
+      || oldConfig.markdownStyle != config.markdownStyle
```

Comment:
MarkdownStyle needs an operator== to prevent this from generating spurious rebuilds.